### PR TITLE
Fix #41

### LINKED ISSUE #41: A player has a position in continuous space
The player's

### PR DESCRIPTION
**Pull request by OggyOggelito**
Fix  #41

### CODE REVIEW

File: player.cpp [hunk 1]
```diff
@@ -12,10 +16,12 @@ void test_player() {
...
     const player player_g;
-    const int expected_player_score = 0;
-    const int player_score = player_g.get_score();
+    const int expected_player_score{0};
+    const int player_score{player_g.get_score()};
     //check player score
     assert(expected_player_score == player_score);
-
-
+    const double x{player_g.get_x()};
```
Comment (TheoPannetier) [inline player.cpp:23]:
```suggestion
    // 41 A player has a position in continuous space
    const double x{player_g.get_x()};
```

File: player.cpp [hunk 2]
```diff
@@ -12,10 +16,12 @@ void test_player() {
...
+    const double y{player_g.get_y()};
```
Comment (TheoPannetier) [inline player.cpp:24]: That's a very smart, concise way to ensure get_x returns a double!

File: player.cpp [hunk 3]
```diff
@@ -1,6 +1,10 @@
 #include "player.h"
 #include <cassert>
-player::player(): m_score{0} {
+player::player() :
+  m_score{0},
+  m_x{0.0},
+  m_y{0.0}
+{
```
Comment (TheoPannetier) [inline player.cpp:7]:
```suggestion
player::player() : m_score{0}, m_x{0.0}, m_y{0.0} {
```

Comment (TheoPannetier) [reply player.cpp:7]: Would linter like this better?

File: player.cpp (hunk 1 above)
Comment (TheoPannetier) [reply player.cpp:23]: I'd like to keep a future reference to the original issue

**Review by TheoPannetier — CHANGES_REQUESTED** [review]
Great work, I suggest adding a reference to the issue to help future contributors get back to it, and maybe a fix for linter?

File: player.cpp (hunk 3 above)
Comment (richelbilderbeek) [reply player.cpp:7]: I think not, let's see what happens :-)

File: player.cpp (hunk 2 above)
Comment (richelbilderbeek) [reply player.cpp:25]: Haha, we knew :sunglasses:

**Comment by richelbilderbeek** [comment]
@TheoPannetier what do you think now? Thanks already for the review!

**Comment by richelbilderbeek** [comment]
Ah, let's resolve those conflicts :-)

**Pull request merged**